tests/tests_api/test_student.py

```diff
@@ -499,11 +506,74 @@ def test_dashboard_student_went_on_leave(api_client, availability_slots, timesta
         availability_slots=availability_slots,
     )
     response = api_client.post(
-        f"/api/dashboard/students/{student.personal_info.id}/went_on_leave/",
+        f"{endpoint}{student.personal_info.id}/went_on_leave/",
     )
     student.refresh_from_db()
     assert response.status_code == status.HTTP_204_NO_CONTENT
     log_event: StudentLogEvent = StudentLogEvent.objects.filter(student_id=student.pk).last()
     assert log_event.type == StudentLogEventType.GONE_ON_LEAVE
     assert student.project_status == StudentProjectStatus.ON_LEAVE
     assert_date_time_with_timestamp(log_event.date_time, timestamp)
+
+
+class TestDashboardStudentReturnedFromLeave:
+    @pytest.mark.parametrize(
+        "endpoint",
+        [
+            "/api/dashboard/students/",
+            "/api/students/",
+        ],
+    )
+    def test_no_group(self, api_client, availability_slots, timestamp, endpoint):
+        student = baker.make(
+            Student,
+            make_m2m=True,
+            _fill_optional=True,
+            project_status=StudentProjectStatus.LEFT_PREMATURELY,
```

Comment:
I know this is irrelevant right now, but I'd rather either use some different status (because `LEFT_PREMATURELY` is for the situation when the person already left the project, so they wouldn't go on leave) or let faker choose some random status. The reason why I don't want this `LEFT_PREMATURELY` here is that if we introduce some logic for checking status changes, this test here will break for no apparent reason.

Reply:
Okay, removed it